akka-remote/src/test/scala/akka/remote/artery/tcp/ssl/RotatingKeysSSLEngineProviderSpec.scala

```diff
@@ -272,7 +272,21 @@ abstract class RotatingKeysSSLEngineProviderSpec(extraConfig: String)
     systemsToTerminate.foreach { systemToTerminate =>
       system.log.info(s"Terminating $systemToTerminate...")
       val now = System.nanoTime()
-      Await.result(systemToTerminate.terminate(), 8.seconds.dilated)
+      try {
+        Await.result(systemToTerminate.terminate(), 8.seconds.dilated)
```

Comment:
better to use `shutdown` in testkit, which will print some stuff if shutdown fails